src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/service/QueueAdminServiceTest.kt

```diff
@@ -0,0 +1,155 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.service
```

Comment:
This has been copied from key worker. I'll make this more relevant in the next PR